docs/systemd-timers.md

```diff
@@ -0,0 +1,57 @@
+# Running CalendarSync periodically using systemd
+
+To run CalendarSync periodically / automatically on specific times using systemd, two files are necessary.
+
+- A [service unit](https://www.freedesktop.org/software/systemd/man/systemd.service.html) file, which we will call: `CalendarSync.service`
```

Comment:
Nitpick, `, which we will call` is somewhat redundant. Replacing it with `called` or removing it entirely would also be fine.

Reply:
removed 👍🏼